src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -678,6 +681,17 @@ function getSecondaryReportActions({
         options.push(CONST.REPORT.SECONDARY_ACTIONS.CHANGE_WORKSPACE);
     }
 
+    // @todo we will remove checking whether current manager is admin in PR #68353
+    // When report manager is not the policy admin and current user is policy admin, allow changing the approver
+    if (
+        !isMemberPolicyAdmin(policy, getLoginByAccountID(report.managerID ?? CONST.DEFAULT_NUMBER_ID)) &&
+        isExpenseReportUtils(report) &&
+        isProcessingReportUtils(report) &&
+        isPolicyAdmin(policy)
```

Comment:
We forgot to check if the approval is enabled or not https://github.com/Expensify/App/issues/76655.